src/didactUriCompletionItemProvider.ts

```diff
@@ -18,271 +18,172 @@
 'use strict';
 
 import * as vscode from 'vscode';
-import * as extensionFunctions from './extensionFunctions';
-import {DidactUri} from './didactUri';
+import * as path from 'path';
+import * as fs from 'fs';
```

Comment:
Any reasons for not using VS Code [filesystem](https://code.visualstudio.com/api/references/vscode-api#FileSystem) API?

Reply:
The 'fs' API is being used in many parts of Didact (didactUriCompletionItemProvider, didactWebView, extensionFunctions, scaffoldUtils, utils) and we can definitely investigate moving over to the VS Code FS API in another issue, but I don't think it's relevant to this particular change. Please open a new JIRA to investigate that change.

Reply:
there is already an issue https://issues.redhat.com/browse/FUSETOOLS2-766
avoiding to use it in new code, avoids to make the debt bigger but fine.